pkg/config/config.go

```diff
@@ -19,7 +20,8 @@ const (
 // OdoSettings holds all odo specific configurations
 type OdoSettings struct {
 	// Controls if an update notification is shown or not
-	UpdateNotification *bool `json:"updatenotification,omitempty"`
+	UpdateNotification *bool   `json:"updatenotification,omitempty"`
+	Prefix             *string `json:"prefix,omitempty"`
```

Comment:
add a single line comment for the parameter

Reply:
Ack